tests/test_telemetry.py

```diff
@@ -369,3 +369,64 @@ def test_app_dependencies_loaded(self):
         for dependency, seen in seen_loaded_dependencies.items():
             if not seen:
                 raise Exception(dependency + " not recieved in app-dependencies-loaded message")
+
+    def setup_app_integrations_change(self):
+        self.r = weblog.get("/enable_integration")
+
+    @irrelevant(library="php", reason="Not possible to dynamically load integrations on PHP")
+    @irrelevant(library="cpp")
+    @irrelevant(library="golang")
+    @irrelevant(library="ruby")
+    @irrelevant(
```

Comment:
When an endpoint is missing on weblog, we skip the test using `missing_feature`.